TmdbEasy/RequestHandler.cs

```diff
@@ -1,23 +1,26 @@
 ﻿using System.Threading.Tasks;
+using TmdbEasy.Configurations;
 using TmdbEasy.Interfaces;
 
 namespace TmdbEasy
 {
     public class RequestHandler : IRequestHandler
     {
         private readonly ITmdbEasyClient _client;
+        private readonly TmdbEasyOptions _options;
 
-        public RequestHandler(ITmdbEasyClient client)
+        public RequestHandler(ITmdbEasyClient client, TmdbEasyOptions options)
```

Comment:
This wont work with DI out of the box. In this case the options are not instantiated by the DI system so best to not include it in the constructor param. The options should be provided to the RequestHandler by the _client. So we need a method on _client called GetOptions() that passes on the options. this simplifies the constructor

Reply:
Okey, i will add an internal TmdbEasyOptions getter to the TmdbEasyClient class.

Reply:
Once this is in, I will merge, thanks for all the work on this

Reply:
Couldn't be internal because the `client` parameter is a public interface :disappointed: